packages/create-llama/create-app.ts

```diff
@@ -88,20 +88,41 @@ export async function createApp({
   };
 
   if (frontend) {
+    let successfulInstallation = true;
+
     // install backend
     const backendRoot = path.join(root, "backend");
     await makeDir(backendRoot);
-    await installTemplate({ ...args, root: backendRoot, backend: true });
+    try {
+      await installTemplate({
+        ...args,
+        root: backendRoot,
+        backend: true,
+      });
+    } catch (error) {
+      successfulInstallation = false;
```

Comment:
Here we have two errors:
1. Code generation fails
2. Install dependencies fails

If 1. I guess we don't want to continue installing the frontend?

Reply:
I thought about this, but our code currently is not easy to do that because: 

we do not separate code generation with dependency installation (for frontend and backend), but we can do that by introduce two types of error then handle it accordingly (but i think this one is not kind of our code pattern).
so i just consider the errors as one type and continue installing the frontend.

do you have any suggestion?

Reply:
Makes sense. The problem here is that we might have an error that requires user interaction (i.e. installing poetry).
Can we easily show the reason for the error at the end (before the process.exit), e.g. by doing `installationError = error`?

Reply:
Nice suggestion, i'll add it